static/manual_body.Rmd

```diff
@@ -1,237 +0,0 @@
----
```

Comment:
Thanks, I've kept that `Rmd` file.  It looks pretty static so I thought it was just html, apologies.